### PR TITLE
MAINTAINERS.md: Introduce initial governance

### PR DESCRIPTION
**Pull request by lilic**
@brancz @metalmatze @paulfantom @pgier @s-urbaniak @simonpasquier @squat @vsliouniaev @coderanger please have a look, in case I missed someone or you feel like someone should not be there. Also check your info and add details about k8s slack and emails so I update the PR, thanks! :) 🎉

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+|-----------------------|------------------------|--------------------------|---------------------------------------------------|-------------------|
+| Frederic Branczyk     | fbranczyk@gmail.com    | `@brancz`                | [@brancz](https://github.com/brancz)              | Red Hat           |
+| Lili Cosic            | coscilili@gmail.com    | `@lili`                  | [@lilic](https://github.com/lilic)                | Red Hat           |
+| Matthias Loibl        | mail@matthiasloibl.com | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)      | Red Hat           |
+| Paweł Krupa           | paulfantom@gmail.com   | ``                       | [@paulfantom](https://github.com/paulfantom)      | Red Hat           |
+| Paul Gier             | pgier@redhat.com       | ``                       | [@pgier](https://github.com/pgier)                | Red Hat           |
+| Sergiusz Urbaniak     | mail@matthiasloibl.com | ``                       | [@s-urbaniak](https://github.com/s-urbaniak)      | Red Hat           |
```
Comment (s-urbaniak) [inline MAINTAINERS.md:10]: Kubernets Slack alias is `@sur`

Comment (s-urbaniak) [reply MAINTAINERS.md:10]: Email is sergiusz.urbaniak@gmail.com

**Comment by s-urbaniak** [comment]
thank you!

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## How to be maintainer?
+
+Any contributor that shows effort, consistentcy and willingness in maintaining this repository will be invited to join the Maintainers team.
+
+Open Source is all about the trust, which is the key factor in decision to add write permissions.
+
+Reach us if you have any questions on how to join maintainer team.
+
+## Initial authors
```
Comment (paulfantom) [inline MAINTAINERS.md:36]: I believe in the case of thanos maintainers file this section is about initial authors of the project.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+| Frederic Branczyk     | fbranczyk@gmail.com    | `@brancz`                | [@brancz](https://github.com/brancz)              | Red Hat           |
+| Lili Cosic            | coscilili@gmail.com    | `@lili`                  | [@lilic](https://github.com/lilic)                | Red Hat           |
+| Matthias Loibl        | mail@matthiasloibl.com | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)      | Red Hat           |
+| Paweł Krupa           | paulfantom@gmail.com   | ``                       | [@paulfantom](https://github.com/paulfantom)      | Red Hat           |
+| Paul Gier             | pgier@redhat.com       | ``                       | [@pgier](https://github.com/pgier)                | Red Hat           |
+| Sergiusz Urbaniak     | mail@matthiasloibl.com | ``                       | [@s-urbaniak](https://github.com/s-urbaniak)      | Red Hat           |
+| Simon Pasquier        |                        | ``                       | [@simonpasquier](https://github.com/simonpasquier)| Red Hat           |
```
Comment (simonpasquier) [inline MAINTAINERS.md:11]: My Slack nickname is `@SimonPasquier` and my email is pasquier.simon@gmail.com.

File: MAINTAINERS.md (hunk 2 above)
Comment (lilic) [reply MAINTAINERS.md:36]: I see, I can change it to just reference this was copied or based on thanos file?

File: code-of-conduct.md [hunk 4]
```diff
@@ -33,29 +33,8 @@ This code of conduct applies both within project spaces and in public spaces
...
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting a project maintainer, Brandon Philips
-<brandon.philips@coreos.com>, and/or Rithu John <rithu.john@coreos.com>.
+reported by contacting a project maintainer listed in the MAINTAINERS.md file. 
 
 This Code of Conduct is adapted from the Contributor Covenant
 (http://contributor-covenant.org), version 1.2.0, available at
 http://contributor-covenant.org/version/1/2/0/
-
-### CoreOS Events Code of Conduct
```
Comment (lilic) [inline code-of-conduct.md:43]: I removed this as we do not have events. We can add if this is ever the case.

File: MAINTAINERS.md (hunk 2 above)
Comment (lilic) [reply MAINTAINERS.md]: Fixed.

**Comment by lilic** [comment]
I also edited the code of contact as it was referencing coreos folks. We should one day review it and perhaps adopt a better one if it exists?

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+## Core Maintainers of this repository
+
+| Name                  | Email                       | Kubernetes Slack         | GitHub                                            | Company           |
+|-----------------------|-----------------------------|--------------------------|---------------------------------------------------|-------------------|
+| Frederic Branczyk     | fbranczyk@gmail.com         | `@brancz`                | [@brancz](https://github.com/brancz)              | Red Hat           |
+| Lili Cosic            | coscilili@gmail.com         | `@lili`                  | [@lilic](https://github.com/lilic)                | Red Hat           |
+| Matthias Loibl        | mail@matthiasloibl.com      | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)      | Red Hat           |
+| Paweł Krupa           | paulfantom@gmail.com        | ``                       | [@paulfantom](https://github.com/paulfantom)      | Red Hat           |
```
Comment (lilic) [inline MAINTAINERS.md:8]: @paulfantom can you tell me your slack k8s name, thanks!

File: MAINTAINERS.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+| Paul Gier             | pgier@redhat.com            | ``                       | [@pgier](https://github.com/pgier)                | Red Hat           |
+| Sergiusz Urbaniak     | sergiusz.urbaniak@gmail.com | `@sur`                   | [@s-urbaniak](https://github.com/s-urbaniak)      | Red Hat           |
+| Simon Pasquier        | pasquier.simon@gmail.com    | `@SimonPasquier`         | [@simonpasquier](https://github.com/simonpasquier)| Red Hat           |
+| Lucas Servén Marín    | lserven@gmail.com           | `@squat`                 | [@squat](https://github.com/squat)                | Red Hat           |
+| Vasily Sliouniaev     |                             | ``                       | [@vsliouniaev](https://github.com/vsliouniaev)    | Walmart Labs      |
```
Comment (lilic) [inline MAINTAINERS.md:13]: @vsliouniaev can you provide your email and slack handle, thanks! :)

File: MAINTAINERS.md (hunk 5 above)
Comment (paulfantom) [reply MAINTAINERS.md:8]: It's `paulfantom` (everywhere :D )

File: MAINTAINERS.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+## Triage
+
+We also have some nice souls that help triaging issues and PRs. See [here](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) for details about the role's permission.
+
+Full list of triage people is displayed below:
+
+| Name                  | Slack                    | GitHub                                                     | Company         |
+|-----------------------|--------------------------|------------------------------------------------------------|-----------------|
+| Noah Kantrowitz       | `@coderanger`            | [@coderanger](https://github.com/coderanger                | Geomagical/IKEA |
```
Comment (coderanger) [inline MAINTAINERS.md:25]: Missing a close `)` on the link. Otherwise 👍

File: README.md [hunk 8]
```diff
@@ -195,6 +195,10 @@ auto-generated. E.g. `bundle.yaml` originates from the _Jsonnet_ files in
...
 If you find a security vulnerability related to the Prometheus Operator, please
 do not report it by opening a GitHub issue, but instead please send an e-mail to
-the maintainers of the project found in the [OWNERS](OWNERS) file.
+the maintainers of the project found in the [MAINTAINERS.md](MAINTAINERS.md) file.
 
 [operator-vs-kube]: https://github.com/prometheus-operator/prometheus-operator/issues/2510#issuecomment-476692399
+
+## Acknowledgements
+
+prometheus-operator organization logo was created and contributed by [Bianca Cheng Costanzo](https://github.com/bia).
```
Comment (lilic) [inline README.md:204]: @bia let me know if this looks good to you :)

**Comment by s-urbaniak** [comment]
lgtm :+1:

**Pull request merged**